### PR TITLE
Removed EventChunkResource

### PR DESCRIPTION
**Pull request by johan-bjareholt**
- Removed EventChunkResource
- Moved 301 logs to debug

### CODE REVIEW

File: aw_server/log.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def log(self, levelname, message, *args):
         msg = args[0]
         code = args[1]
 
-        if code in [200, 304]:
+        if code in [200, 301, 304, 404]:
```
Comment (ErikBjare) [inline aw_server/log.py]: What what, why is 404 in debug?

Comment (johan-bjareholt) [inline aw_server/log.py]: Why would is be interesting to see 404's in logs when not in --testing?

Comment (ErikBjare) [inline aw_server/log.py]: What? It'd be interesting if a resource on the server was not found. If we get 404s somewhere we should eliminate them.

Comment (johan-bjareholt) [inline aw_server/log.py]: Alright, switched back

**Comment by johan-bjareholt** [comment]
@ErikBjare Totally forgot about this, should i merge?

**Comment by ErikBjare** [comment]
@johan-bjareholt I don't think 301 errors should be ignored. They are likely caused by semi-wrong paths in aw-client: https://github.com/ActivityWatch/aw-client/pull/14

**Comment by johan-bjareholt** [comment]
Added back 301 log messages

**Pull request merged**